examples/pylab_examples/demo_annotation_box.py

```diff
@@ -65,6 +66,7 @@
     arr_lena = read_png(fn)
 
     imagebox = OffsetImage(arr_lena, zoom=0.2)
+    imagebox.image.axes = ax
```

Comment:
Since you are touching this file could you change the `arr_lena` name to something else? It really does not make any sense anymore.